packages/preview/qooklet/0.1.0/README.md

```diff
@@ -0,0 +1,51 @@
+# Qooklet
+
+A quick start template for scientific booklets.
+
+## Features
+
+- Math Equation
+  - auto numbering based on chapter
+  - link quote to source
+- Figure
+  - auto numbering based on chapter
+  - subfigure (using [subpar](https://github.com/tingerrr/subpar))
+- Table
+  - read as three-line table (`#ktable(data, k, inset: 0.3em)`)
+  - read from .xlsx (using [rexllent](https://github.com/hongjr03/typst-rexllent))
+- Code Block
+  - highlighting (using [coldly](https://github.com/Dherse/codly))
+  - read code block (`#code(text, lang: "python", breakable: true, width: 100%)`)
+- Theorem
+  - auto numbering based on chapter (using [ctheorems](https://github.com/sahasatvik/typst-theorems))
+  - multilingual (using [linguify](https://github.com/typst-community/linguify))
+
+## Get Started
+
+```typst
+#import "@preview/qooklet:0.1.0": *
+#show: doc => conf(
+  title: "Bellman Eqation",
+  author: "ivaquero",
+  header-cap: "Reinforcement Learning",
+  footer-cap: "ivaquero",
+  outline-on: false,
+  doc,
+)
+```
+
+## Get Started
```

Comment:
It would be better to distinguish between this section and the previous one by giving them different titles. I also think this section should have some instructions on how to make the package available in the `@local` namespace.

Reply:
Yes, you are right, I copied wrong section here.